components/jupyter-web-app/kubeflow_jupyter/default/templates/layout.html

```diff
@@ -1,45 +1,47 @@
 <!DOCTYPE html>
 <html>
 <head>
-  <!-- Required meta tags -->
-  <meta charset="utf-8">
-  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <!-- Required meta tags -->
```

Comment:
The indentation settings seem to be different here and they really seem to be out of whack in the templates below. You may need to disable auto-formatting on save temporarily to avoid changing more than just the relevant sections as it's somewhat tough to see what changed.

Reply:
I see, will do